src/AppBundle/Controller/SimpleEditCounterController.php

```diff
@@ -57,11 +58,15 @@ public function resultAction($project, $username)
 
         $username = ucfirst($username);
 
-        $dbValues = $lh->databasePrepare($project, "SimpleEditCounter");
+        $projectData = ProjectRepository::getProject($project, $this->container);
 
-        $dbName = $dbValues["dbName"];
-        $wikiName = $dbValues["wikiName"];
-        $url = $dbValues["url"];
+        if (!$projectData->exists()) {
+            $this->addFlash("notice", ["invalid-project", $project]);
+            return $this->redirectToRoute("SimpleEditCounter");
+        }
+
+        $dbName = $projectData->getDatabaseName();
+        $url = $projectData->getUrl();
```

Comment:
Speaking of variable names, I think this one should be `$projectUrl` (like it is in some other places), to differentiate from other URLs.